src/components/Article.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({ article }) => {
```

Comment:
Article component should accept 3 fields in props: title, date and next, not full article as separate object